libqtile/widget/generic_poll_text.py

```diff
@@ -61,6 +61,10 @@ def __init__(self, **config):
         if self.data and not isinstance(self.data, str):
             self.data = json.dumps(self.data).encode()
 
+    def _configure(self, qtile, bar):
+        base.ThreadPoolText._configure(self, qtile, bar)
```

Comment:
Why not just do this in `ThreadPoolText`? That's the class that defines the `force_update` method.